packages/core/tests/universal-tokens.js

```diff
@@ -0,0 +1,102 @@
+import createCss from '../src/index.js'
+
+describe('Tokens', () => {
+	test('Authors can define and use a relative token', () => {
+		{
+			const { global, toString } = createCss({
+				theme: {
+					colors: {
+						red: 'tomato',
+					},
+				},
+			})
+
+			global({
+				article: {
+					color: '$red',
+				},
+			})()
+
+			expect(toString()).toBe(`:root{--colors-red:tomato;}` + `article{color:var(--colors-red);}`)
+		}
+
+		{
+			const { global, toString } = createCss({
+				theme: {
+					shadows: {
+						red: 'tomato',
+					},
+				},
+			})
+
+			global({
+				article: {
+					boxShadow: '0 0 0 1px $red',
+				},
+			})()
+
+			expect(toString()).toBe(`:root{--shadows-red:tomato;}` + `article{box-shadow:0 0 0 1px var(--shadows-red);}`)
+		}
+	})
+
+	test('Authors can use an absolute token', () => {
+		{
+			const { global, toString } = createCss({
+				theme: {
+					colors: {
+						red: 'tomato',
+					},
+				},
+			})
+
+			global({
+				article: {
+					boxShadow: '0 0 0 1px $colors$red',
+				},
+			})()
+
+			expect(toString()).toBe(`:root{--colors-red:tomato;}` + `article{box-shadow:0 0 0 1px var(--colors-red);}`)
+		}
+
+		{
+			const { global, toString } = createCss({
+				theme: {
+					colors: {
+						red: 'tomato',
+					},
+				},
+			})
+
+			global({
+				article: {
+					boxShadow: '0 0 0 1px $colors$red',
+				},
+			})()
+
+			expect(toString()).toBe(`:root{--colors-red:tomato;}` + `article{box-shadow:0 0 0 1px var(--colors-red);}`)
+		}
+	})
+
+	test('Authors can use a negative token', () => {
+		{
+			const { global, toString } = createCss({
+				theme: {
+					sizes: {
+						sp1: '10px',
+						sp2: '20px',
+						sp3: '30px',
+					},
+				},
+			})
+
+			global({
+				article: {
+					marginLeft: '-$sizes$sp1',
+					width: '$sp1',
```

Comment:
Is it worth adding a test for a simple negative token, without prefixing the scale?

Reply:
Yes it is. Thanks for the call out. Added in https://github.com/modulz/stitches/pull/411/commits/f4b27d37d4d83dd42e417a2a986b2fc682e492a8